wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/model/impl/devfile/ComponentImpl.java

```diff
@@ -346,6 +413,7 @@ public boolean equals(Object o) {
   public int hashCode() {
 
     return Objects.hash(
+        generatedId,
         getName(),
```

Comment:
In `equals` method you changed usage of `getter` to field. Please do here the same, or revert changes in `equals`